samples/caf_sensor_manager/src/modules/sensor_sim_ctrl.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+#include <drivers/sensor_sim.h>
+
+#include "sensor_sim_ctrl_def.h"
+
+#define MODULE sensor_sim_ctrl
+#include <caf/events/module_state_event.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(MODULE);
+
+static int set_wave(void)
+{
+	const struct sim_wave *w = &sim_signal_params.wave;
+	int err = sensor_sim_set_wave_param(DEVICE_DT_GET(DT_NODELABEL(sensor_sim)),
+					    sim_signal_params.chan,
+					    &w->wave_param);
+
+	if (err) {
+		LOG_ERR("Cannot set simulated accel params (err %d)", err);
+		return err;
+	}
+	return 0;
+}
+
+static void verify_wave_params(void)
+{
+	const struct sim_wave *w = &sim_signal_params.wave;
+
+	__ASSERT_NO_MSG(w->label != NULL);
+	__ASSERT_NO_MSG(w->wave_param.type < WAVE_GEN_TYPE_COUNT);
+	__ASSERT_NO_MSG((w->wave_param.period_ms > 0) ||
+			(w->wave_param.type == WAVE_GEN_TYPE_NONE));
+	ARG_UNUSED(w);
+
+}
+
+static int init(void)
+{
+	if (IS_ENABLED(CONFIG_ASSERT)) {
+		verify_wave_params();
+	}
+	return set_wave();
```

Comment:
add empty line after `}` too.

Reply:
I think @emob-nordic wanted to add empty line after line 48 not before line 51 - can you move it?